### ISSUE TITLE
Yahoo Finance data (OHLC) is entirely split adjusted, AdjustOHLC should only be adjusting for dividends at this point.

### ISSUE THREAD

**Issue by gmahjub** [joshuaulrich/quantmod#218]
### Description

Yahoo finance data is now entirely split adjusted. AdjustOHLC should no longer adjust for splits as doing it again would double adjust for splits. Verified this by writing my own function to calculate adjustment ratios using only dividends, and was able to match adjusted close prices with Yahoo's adjusted close prices. Also, you will see that Yahoo has a note at the bottom of the Historical Data tab that says all data is split adjusted, and adjusted close is split and dividend adjusted.

**Comment by LuisSousaSilva** [comment]
I didn't run anything code wise but. on the historical Tab i still see "*Close price adjusted for splits.**Adjusted close price adjusted for both dividends and splits."

OHL seem to still be raw.

**Comment by Javdat** [comment]
All of OHLC is adjusted for splits but not for dividends:

```
library(quantmod)
appl.yahoo <- getSymbols("AAPL", from = "2014-01-01", auto.assign = FALSE)
uvxy.yahoo <- getSymbols("UVXY", from = "2007-01-01", auto.assign = FALSE)

# APPLE split 06/09/2014	7 for 1
appl.yahoo["2014-06-06"]
# UVXY 07/17/2017	1 for 4 (no dividends at all)
uvxy.yahoo["2017-07-14"]
```

Response:
```
> appl.yahoo["2014-06-06"]
           AAPL.Open AAPL.High AAPL.Low AAPL.Close AAPL.Volume AAPL.Adjusted
2014-06-06  92.84286  93.03714 92.06715   92.22429    87484600      86.14337
> uvxy.yahoo["2017-07-14"]
           UVXY.Open UVXY.High UVXY.Low UVXY.Close UVXY.Volume UVXY.Adjusted
2017-07-14      35.6     35.64     33.8      34.24    30277700         34.24
```

Difference in AAPL.Adjusted and the rest is due to dividends only (difference would have been much higher in case of split). In case of UVXY, it clear that there is split adjustment.

**Comment by joshuaulrich** [comment]
I am not inclined to change the default arguments to other functions because of changes to Yahoo data. I realize Yahoo is the default data source for `getSymbols()`, but it has changed frequently over the past year and the quality is much poorer than it had been.  I have strongly considered changing the default to another source.

When they first made major changes in May-2017, I tried to make the output of `getSymbols()` the same as it had been before the change. The adjustment of the data changed a few times over the next month. At one point, OHL prices were also adjusted.  At another point, they were unadjusted.  The adjusted close also flipped from being only split adjusted to split and dividend adjusted.  Because of that, I made `getSymbols()` simply return whatever Yahoo sent as-is.  I don't want to be responsible for updating data or default arguments whenever they change something.
